.github/workflows/publish-stage.yaml

```diff
@@ -43,7 +39,7 @@ jobs:
         with:
           switches: -avzr --delete
           path: build/
-          remote_path: ${{ secrets.AWS_STAGE_PUBLISH_PATH }}
+          remote_path: ${{ secrets.AWS_STAGE_PUBLISH_PATH_UNSUPPORTED }}/0.25
```

Comment:
I've already added this new key to the project secrets (and to the Keeper record that contains these secrets).